bridge-cli/src/omni_connector_command.rs

```diff
@@ -20,170 +20,209 @@ use crate::{combined_config, CliConfig, Network};
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to finalize the transfer on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(
+            short,
+            long,
+            help = "Storage deposit actions. Format: token_id1:account_id1:amount1,token_id2:account_id2:amount2,..."
+        )]
+        storage_deposit_actions: Vec<String>,
+        #[clap(short, long, help = "VAA from init transfer transaction")]
         vaa: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Deploy a token on EVM")]
     EvmDeployToken {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to deploy the token on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(
+            short,
+            long,
+            help = "Transaction hash of the LogMetadata call on other chain"
+        )]
         tx_hash: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Bind a token on EVM")]
     EvmBindToken {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to bind the token on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(short, long, help = "VAA from DeployToken call")]
         tx_hash: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Initialize a transfer on EVM")]
     EvmInitTransfer {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to initialize the transfer on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Token to transfer")]
         token: String,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Amount to transfer")]
         amount: u128,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Recipient address on the other chain")]
         receiver: String,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Fee to charge for the transfer")]
         fee: u128,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Native fee to charge for the transfer")]
         native_fee: u128,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Finalize a transfer on EVM")]
     EvmFinTransfer {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to finalize the transfer on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(
+            short,
+            long,
+            help = "Transaction hash of the sign_transfer call on NEAR"
+        )]
         tx_hash: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
 
+    #[clap(about = "Initialize a transfer on Solana")]
     SolanaInitialize {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Keypair (private_key) for the program")]
```

Comment:
Can also be a .json file according to latest update

Reply:
Perhaps let's also specify that for a string value it must be in Base58
"Solana keypair in Base58 or path to a .json keypair file"